Tools/autotest/common.py

```diff
@@ -766,20 +788,98 @@ def wait_pitch(self, pitch, accuracy, timeout=30):
         self.progress("Failed to attain pitch %d" % pitch)
         raise WaitPitchTimeout()
 
-    def wait_heading(self, heading, accuracy=5, timeout=30):
+    def wait_heading(self, heading, accuracy=5, maintain_target_time=0, timeout=30, the_function=None):
```

Comment:
use `None` for `maintain_target_time` - even if it's a bit more code further down.